src/shibuya.js

```diff
@@ -28,24 +28,24 @@ const main = async () => {
     const proxyOnShibuya = shibuyaHelper.getProxyAccount(TURING_PARA_ID, shibuyaAddress);
 
     console.log(`\na) Add a proxy of Turing (paraId:${TURING_PARA_ID}) for Alice on Shibuya ...\n Proxy address: ${proxyOnShibuya}\n`);
-    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair);
+    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair); 
 
     console.log('\nb) Topping up the proxy account on Shibuya with SBY ...\n');
-    const transferSBY = shibuyaHelper.api.tx.balances.transfer(proxyOnShibuya, '1000000000000000000000');
+    const transferSBY = shibuyaHelper.api.tx.balances.transfer(proxyOnShibuya, '1000000000000000000000'); 
     await sendExtrinsic(shibuyaHelper.api, transferSBY, keyPair);
 
     console.log('\n2. One-time proxy setup on Turing');
     const proxyOnTuring = turingHelper.getProxyAccount(SHIBUYA_PARA_ID, turingAddress);
 
     console.log(`\na) Add a proxy of Shibuya (paraId:${SHIBUYA_PARA_ID}) for Alice on Turing ...\nProxy address: ${proxyOnTuring}\n`);
-    await sendExtrinsic(turingHelper.api, turingHelper.api.tx.proxy.addProxy(proxyOnTuring, 'Any', 0), keyPair);
+    await sendExtrinsic(turingHelper.api, turingHelper.api.tx.proxy.addProxy(proxyOnTuring, 'Any', 0), keyPair); 
 
     console.log('\nb) Topping up the proxy account on Turing with TUR ...\n');
     const transferExtrinsic = turingHelper.api.tx.balances.transfer(proxyOnTuring, '10000000000000');
-    await sendExtrinsic(turingHelper.api, transferExtrinsic, keyPair);
+    await sendExtrinsic(turingHelper.api, transferExtrinsic, keyPair); 
```

Comment:
Restart the blockchains, and test whether TUR is needed at all. I suspect that we don’t need this any more.

Reply:
Not needed any more, and deleted.